package.json

```diff
@@ -28,8 +28,8 @@
   "license": "MIT",
   "devDependencies": {
     "fastify": "^2.0.0",
-    "fastify-jwt": "^0.9.0",
-    "fastify-leveldb": "^0.3.0",
+    "fastify-jwt": "^0.9.3",
+    "fastify-leveldb": "^1.0.0",
```

Comment:
I don't think this change is up to this PR, since it should only to clarify some pieces of code